### PR TITLE
[ch190] Add dataset metadata schema

### PR DESCRIPTION
**Pull request by seerocode**
Story details: https://app.clubhouse.io/stanford-computational-policy-lab/story/190

### CODE REVIEW

File: api/database.py [hunk 1]
```diff
@@ -287,6 +304,26 @@ def create_dummy_data(session):
...
+    category_non_binary = Category(id='51349e29-290e-4398-a401-5bf7d04af75e', description='Non-binary gender',
+                                    category='gender', category_value='non-binary')
+    category_women = Category(id='0034d015-0652-497d-ab4a-d42b0bdf08cb', description='Women gender',
+                                    category='gender', category_value='women')
+    category_men = Category(id='d237a422-5858-459c-bd01-a0abdc077e5b', description='Men gender',
+                                    category='gender', category_value='men')
+    category_non_binary.targets.append(target_non_binary)
+    category_women.targets.append(target_women)
+    category_men.targets.append(target_men)
+
```
Comment (nmanduj1) [inline api/database.py:333]: ty for adding dummy data!

**Review by nmanduj1 — APPROVED** [review]

**Comment by seerocode** [comment]
@nmanduj1 the tests are passing here but not in the subsequent PRs I opened that are based off this. I'm not sure what's going on.

**Comment by seerocode** [comment]
I need to add additional gender identities to dummy data and then I will merge.

File: api/database.py (hunk 1 above)
Comment (seerocode) [reply api/database.py:333]: @nmanduj1 @ameliagoodman @jnu I made the gender identities a little more clear here.

**Review by nmanduj1 — APPROVED** [review]

**Pull request merged**